README.rst

```diff
@@ -65,6 +65,39 @@ Follow the usual CMake build workflow:
     cmake --install _build
 
 
+Obtaining libXC
+---------------
+
+Follow the usual CMake build workflow:
+
+* Clone the official libXC repository and checkout the latest release tag, e.g.
+  ``6.2.2``::
+
+    git clone https://gitlab.com/libxc/libxc.git libxc
+    cd libxc/
+    git checkout 6.2.2
+
+* Configure the project, specify your compilers (e.g. ``gfortran`` and ``gcc``),
+  the install location (e.g. ``$HOME/opt/libxc``) and the build directory
```

Comment:
Probably should mention this needs to match the path above.

Reply:
Please see revised version.